agent/taskresource/cgroup/cgroup.go

```diff
@@ -22,11 +22,11 @@ import (
 	"sync"
 	"time"
 
-	"github.com/aws/amazon-ecs-agent/agent/api"
 	"github.com/aws/amazon-ecs-agent/agent/resources/cgroup"
 	"github.com/aws/amazon-ecs-agent/agent/utils/ioutilwrapper"
 	"github.com/cihub/seelog"
 	"github.com/containerd/cgroups"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
Is this a new dependency? Does this require a dep update?

Reply:
This does need update. As @adnxn mentioned, we can make it track the latest release, which I shall do in another PR.